### PR TITLE
Kills off the master controller

### PR DESCRIPTION
**Pull request by AffectedArc07**
**What does this PR do:**
Kills off the master controller, its long overdue for its death. Jobs controller is now `SSjobs` and the rest of the master controller initialisation stuff is handled elsewhere

# NOTICE
I would like this testmerged before being fully merged. I did test this and it worked for the jobs I tried, but I can not sit there and try every job, or try what happens when jobs are full, etc. There should be no changes to job functionality but this is BYOND so who fucking knows.

**Changelog:**
*inb4 this changelog freaks everybody out*

:cl: AffectedArc07
del: Master Controller
/:cl:

### CODE REVIEW

**Comment by KasparoVy** [comment]
https://www.youtube.com/watch?v=QKSMl7sBNf8

We fullly stoned now :sunglasses:

**Comment by AffectedArc07** [comment]
https://github.com/ParadiseSS13/Paradise/blob/e377bdc2cc356f07c89588b1078edf7226627963/code/controllers/ProcessScheduler/core/processScheduler.dm#L2

>Fully stoned

When I kill this bastard off as well

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -689,11 +689,11 @@ var/list/admin_verbs_ticket = list(
 	if(!check_rights(R_DEBUG))
 		return
 
-	if(air_processing_killed)
```
Comment (Fox-McCloud) [inline code/modules/admin/admin_verbs.dm]: I'll be completely honest, I'm not even sure if this entire proc should exist anymore.

This harkens back to the days of ZAS, where it lagged like hell and caused lots of problem and was prone to breaking (modern ZAS is much better).

Comment (AffectedArc07) [reply code/modules/admin/admin_verbs.dm]: You want me to gut that?

Comment (AffectedArc07) [reply code/modules/admin/admin_verbs.dm]: Done

File: code/game/gamemodes/gameticker.dm [hunk 2]
```diff
@@ -134,7 +134,10 @@ var/round_start_time = 0
 	else
 		src.mode.announce()
 
+	setupfactions()
+	setup_economy()
```
Comment (Fox-McCloud) [inline code/game/gamemodes/gameticker.dm]: Wait....

Comment (AffectedArc07) [reply code/game/gamemodes/gameticker.dm]: That jump list isnt working, can I get file and line? And if it was doing it twice, maybe thats why we are so slow to start

Comment (AffectedArc07) [reply code/game/gamemodes/gameticker.dm]: Diagnosis: I cant raed

**Comment by AffectedArc07** [comment]
Few more bits of testing

Code words generate
image

Economy works
image
image

I *think* factions work
image

**Pull request merged**